subsys/nrf_security/CMakeLists.txt

```diff
@@ -117,8 +117,7 @@ target_compile_definitions(psa_crypto_library_config
 # The name and intent of this comes from TF-M distribution
 add_library(psa_interface INTERFACE)
 
-# In nRF54L this configuration is required for the HUK keys to work
-if(CONFIG_SOC_SERIES_NRF54LX AND (CONFIG_HW_UNIQUE_KEY OR CONFIG_IDENTITY_KEY))
+if(CONFIG_MBEDTLS_ENABLE_BUILTIN_KEYS)
 # Add config files required for PSA crypto interface
```

Comment:
not a requirement for this PR but this file should be fixed and properly indented

Reply:
Oufh, my pre-commit hook is not enabled in this repo. Sorry about that.